roles/network/templates/hostapd/50-hostapd

```diff
@@ -26,7 +26,7 @@ if [ "$interface" = "wlan0" ]; then
     fi
     # spams the logging
     #syslog info "50-iiab set ap0 spam $REASON"
-    if [ -e /sys/class/net/ap0 ] && [ "$reason" = "*CARRIER" ]; then
+    if [ -e /sys/class/net/ap0 ] && [ "$reason" = "CARRIER" ] || [ "$reason" = "NOCARRIER" ]; then
```

Comment:
It looks like parentheses are needed, perhaps something like the following:

```suggestion
    if [ -e /sys/class/net/ap0 ] && ( [ "$reason" = "CARRIER" ] || [ "$reason" = "NOCARRIER" ] ); then
```

Reply:
The above without () has been field tested on .30, 
>May 13 03:10:44 box.lan dhcpcd[431]: wlan0: carrier acquired
May 13 03:10:44 box.lan dhcpcd[431]: wlan0: 50-iiab CARRIER change wlan0
May 13 03:10:44 box.lan dhcpcd[431]: frequency is 6 for carrier
May 13 03:10:44 box.lan dhcpcd[431]: Using 6 for carrier
May 13 03:10:44 box.lan dhcpcd[431]: wlan0: 50-iiab set channel 6
May 13 03:10:44 box.lan dhcpcd[431]: Hostapd set for 6
May 13 03:10:44 box.lan dhcpcd[431]: wlan0: 50-iiab ap0 spam CARRIER
May 13 03:10:44 box.lan dhcpcd-run-hooks[624]: wlan0: 50-iiab ap0 spam CARRIER
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: 50-iiab set ap0 up CARRIER
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: 50-iiab CARRIER change wlan0
May 13 03:10:49 box.lan dhcpcd[431]: frequency is 6 for carrier
May 13 03:10:49 box.lan dhcpcd[431]: Using 6 for carrier
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: 50-iiab set channel 6
May 13 03:10:49 box.lan dhcpcd-run-hooks[672]: wlan0: 50-iiab set channel 6
May 13 03:10:49 box.lan dhcpcd[431]: Hostapd set for 6
May 13 03:10:49 box.lan dhcpcd[431]: DUID 00:01:00:01:25:d8:31:74:dc:a6:32:0d:3a:a8
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: IAID 32:0d:3a:a9
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: adding address fe80::b66a:d4b4:a4ab:4aa6
May 13 03:10:49 box.lan dhcpcd[431]: ipv6_addaddr1: Permission denied
May 13 03:10:49 box.lan dhcpcd[431]: ipv6_addaddr1: Permission denied
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: carrier lost
May 13 03:10:49 box.lan dhcpcd[431]: wlan0: 50-iiab ap0 spam NOCARRIER
May 13 03:10:54 box.lan dhcpcd[431]: wlan0: 50-iiab set ap0 up NOCARRIER

I'll tried the change on .34 works also, so it a question of esthetics.